### DISCUSSION TITLE
Code coverage

### DISCUSSION THREAD

**Discussion by kli4d-o** [MarvinKweyu/ColorDetect#46] [Ideas]
Coverage reports provide metrics that show how deeply the code has been tested. Several tools provide simple ways to visualise such reports, though the one I'm most familiar with is [Codecov](https://about.codecov.io/). I suggest the repository to make use of [Codecov](https://about.codecov.io/) or some other tool providing code coverage visualisation.

**Comment by MarvinKweyu** [comment]
@onyonkaclifford This is a great way to go through it. Can be integrated into `v1.4.7` or the upcoming `v1.5.0`
